simulator.go

```diff
@@ -250,6 +251,20 @@ func (h *simulatorAPIHandler) CheckTimeout() {
 }
 
 func (h *simulatorAPIHandler) terminateContainer(id string, w http.ResponseWriter) {
+
+	_, ok := h.terminating[id]
```

Comment:
This is probably no longer essential. It was added under the assumption that the containers were ending prematurely or being Del'd by the simulator. The scenario where this would be useful would be that the simulator calls for example Delete after a Submitresults.